wpimath/src/main/java/edu/wpi/first/math/geometry/Translation2d.java

```diff
@@ -106,27 +89,7 @@ public Translation2d(Vector<N2> vector) {
    * @return The distance between the two translations.
    */
   public double getDistance(Translation2d other) {
-    return Math.hypot(other.m_x - m_x, other.m_y - m_y);
-  }
-
-  /**
-   * Returns the X component of the translation.
-   *
-   * @return The X component of the translation.
-   */
-  @JsonProperty
-  public double getX() {
-    return m_x;
-  }
```

Comment:
I'll note that the old accessor is annotated with `@JsonProperty` while the new accessor is annotated with `@JsonProperty(required = true, value = "x")` because that's what the record component is annotated with in the class definition. Not sure if that'll be an issue, but worth noting.